ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash -e
+# Setup peer-pods using cloud-api-adaptor on azure
+
+###############################
+# Disable security to allow e2e
+###############################
+
+# Disable security
+oc adm policy add-scc-to-group privileged system:authenticated system:serviceaccounts
+oc adm policy add-scc-to-group anyuid system:authenticated system:serviceaccounts
+oc label --overwrite ns default pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/warn=baseline pod-security.kubernetes.io/audit=baseline
+
+
+####################################
+# Get basic credentials from cluster
+####################################
+
+oc -n kube-system get secret azure-credentials -o json > azure_credentials.json
+AZURE_CLIENT_ID="$(jq -r .data.azure_client_id azure_credentials.json|base64 -d)"
+AZURE_CLIENT_SECRET="$(jq -r .data.azure_client_secret azure_credentials.json|base64 -d)"
+AZURE_TENANT_ID="$(jq -r .data.azure_tenant_id azure_credentials.json|base64 -d)"
+AZURE_SUBSCRIPTION_ID="$(jq -r .data.azure_subscription_id azure_credentials.json|base64 -d)"
+rm -f azure_credentials.json
+AZURE_RESOURCE_GROUP=$(oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.azure.resourceGroupName}')
+az login --service-principal -u "${AZURE_CLIENT_ID}" -p "${AZURE_CLIENT_SECRET}" --tenant "${AZURE_TENANT_ID}"
+
+AZURE_VNET_NAME=$(az network vnet list --resource-group "$AZURE_RESOURCE_GROUP" --query "[].{Name:name}" --output tsv)
```

Comment:
Shellcheck reported 66 nits for this script, e.g. for this line it is complaining about `$AZURE_RESOURCE_GROUP` that should be `${AZURE_RESOURCE_GROUP}`

```
Prefer putting braces around variable references even when not strictly required. [SC2250]
```

https://github.com/kata-containers/kata-containers/actions/runs/13918996299/job/38951635884?pr=10940#step:4:7600